### DISCUSSION TITLE
ONNX runtime is not returing zero vectors

### DISCUSSION THREAD

**Discussion by thejishnunair** [microsoft/onnxruntime#15268] [Other Q&A]
Hey folks,

I have trained a TFIDF model using scikit-learn and converted it into onnx format. Now when I try to do a batch inference using the onnx model, for records which do not match with the vocabulary of the TFIDF model, it is not returning the zero vector. For example, if 1000 records are being as input, the output is of size 997 vectors. 3 records were zero vectors but were not being returned as part of the inference.

Any idea how to resolve this issue?

**Comment by pranavsharma** [comment]
Please file this as an issue and follow the issue template.
